development.md

```diff
@@ -1,52 +1,81 @@
+# Building with Backends
+
+With all required backends installed, `stack build` can build all packages
+listed in `stack.yaml` and is equivalent to;
+
+```
+> stack build persistent persistent-template persistent-sqlite persistent-test
+persistent-mongoDB persistent-mysql persistent-postgresql persistent-redis
+```
+
+I can build a subset of targets excluding those with backends not installed:
```

Comment:
/s/I/You

Also, I think it can be made slightly more clear:

```
You can build specific packages from a list of targets using:
```

Or something like that.